CGAL_Core/include/CGAL/CORE/BigFloat_impl.h

```diff
@@ -137,7 +137,7 @@ BigFloatRep::BigFloatRep(double d) : m(0), err(0), exp(0) {
       exp--;
       stop++;
     }
-#ifdef CORE_DEBUG
+#ifdef CGAL_CORE_DEBUG
     CGAL_assertion (s >= 0);
 #endif
```

Comment:
(That is just a remark, and not a request for a change.)

I find strange that such an assertion is protected by macro `CORE_DEBUG`, knowing that `CGAL_assertion` itself it protected by `NDEBUG` and `CGAL_NDEBUG`.